### ISSUE TITLE
add support for dailyindiegame's "AAA games store"

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#1088]
https://dailyindiegame.com/content_aaastore.html

[Issue created by bb34887: 2018-06-28]
[Last updated on bitbucket: 2018-07-08]

[Comment created by KNRT: 2018-07-08]
Just a fan.

[Comment created by bb34887: 2018-07-08]
Fair enough and thank you for your extensive responses. BTW are you part of ITAD's staff?

[Comment created by KNRT: 2018-07-07]
I'm aware of the first-sale doctrine and I'm not arguing against it.

All I'm saying is that ITAD doesn't list gray market/unauthorized resellers and as per your question I've provided my reasoning as to why it's obvious that DIG is an unauthorized reseller.

ITAD is a price comparison site covering authorized resellers only. In other words, gray market/unauthorized resellers do not fall within its scope.

I can think of several reasons why the inclusion of gray market/unauthorized resellers is a bad idea and the harm it could cause:

- Damage to ITAD reputation - Visitors trust ITAD to provide links to trusted sites. When a customer receives a dodgy key... well that could be the last time they visit ITAD.

- Lost revenue from decreased traffic - ITAD is funded through affiliate links. More visitors = more clicks. The inclusion of gray market/unauthorized resellers on the site would mean that those fancy tables we see on Reddit GameDeals would no longer be allowed to link to ITAD due to their strict policy against anything gray market. I can't speculate as to how much loss this could potentially result in, but considering that Reddit GameDeals is the largest game deals community in the world I don't think it's something ITAD would want to lose as a referrer.

- Bidding wars for visibility - If gray market/unauthorized resellers were allowed then you can be sure that we would see them get into a constant bidding war over who has the lowest price. This would carry over to Enhanced Steam as whoever is displayed as the "Current Lowest Price" on there would be guaranteed increased traffic.

- Data Quality - Let's say some unauthorized reseller buys 5 copies of some bundle and then lists those games for sale at a price which is lower than the current recorded historical low. Having this price now represented as being the historical low for that game isn't exactly an accurate reflection of its pricing. In other words, the data which is the backbone of ITAD is no longer a reliable resource.

- It would likely be in breach of section 4 of the Steam Web API Terms of Use.

[Comment created by bb34887: 2018-07-07]
I take many of your points, but I still don't like the idea that a publisher should be able to control who can re-sell its products. It goes beyond what the copyright laws intended, in fact there is a legal principle against it - called the [first-sale doctrine](https://en.wikipedia.org/wiki/First-sale_doctrine)

Also, since ITAD is a comparison site, including DIG's prices won't do any harm if they are higher than some other site - customers will simply choose the lower price.

[Comment created by KNRT: 2018-07-06]
A Hat in Time, The Escapists 2 and Conan Exiles have been revealed as the early unlock titles for Humble Monthly ($12) August 2018.

~5 hours later all 3 have been added to the DIG store.

- A Hat in Time - $9.95
- The Escapists 2 - $7.45
- Conan Exiles - $14.95

The additional 5 games which were revealed in the full lineup for Humble Monthly ($12) July 2018 have also now appeared on the DIG store ~5 hours after reveal.

- Forts - $4.95
- Titan Quest: Anniversary Edition + Titan Quest: Ragnarok DLC - $4.95
- Shiness: The Lightning Kingdom - $2.95
- Interplanetary: Enhanced Edition - $2.95
- Serial Cleaner - $2.95

[Comment created by KNRT: 2018-07-06]
**1:** An authorized reseller would have access to a wide catalog of games AT ALL TIMES. DIG only has access to a limited catalog of games and ONLY AFTER they go on sale at authorized resellers like Fanatical, IG, Humble, WinGameStore, etc.

I get emails from all the stores about their sales and then the next day (or often even within hours) the same games are added to DIG, but at a much higher price. It doesn't take a genius to put two and two together.

- Fanatical Independence Day Flash Deals (new deal every hour for 24 hours). Next thing we see is that 12 out of those 24 discounted games from Fanatical are on the DIG store at a 2-4 times higher price.

- Fanatical has "Call of Juarez" as their Star Deal for $0.99 and then we immediately see it added to the list over at DIG for $1.99

- Humble store has a sale on SOMA for $4.49 and then we immediately see it added to the list over at DIG for $9.99

- Age of Wonders III shows up for sale over at IndieGala for $5.99 and then we immediately see it show up on DIG store for $14.99

- Humble Monthly ($12) July instant unlocks are announced as being Hearts of Iron IV, Blackwake, and Portal Knights. We then see Hearts of Iron IV added to DIG store for $19.99 and Portal Knights for $7.45

- Burnin' Rubber 5 HD appears as a daily deal on Chrono.gg for $4 and couple hours later is listed on DIG store for $5.95

- Nail'd on sale for $0.99 at Fanatical as part of their Deep Silver sale and couple hours later is listed on DIG store for $1.99

Literally every single game DIG puts in their store is a game that was just included in a bundle or went on sale somewhere else at a much cheaper price. Coincidence? I think not!

Unlike you however, I do think that this kind of re-sale is "dodgy". The consumer ends up paying more and the publisher doesn't even see that money. How is it ok for some random guy in the middle to profit by basically ripping off everyone?

**2:** Authorized resellers have to adhere to certain conditions regarding:

- Duration of sale: Set by publisher, not run indefinitely until sold out like DIG does

- Refunds:  Authorized resellers are provided with tools, via an API, using which they can cancel keys and issue refunds. In many parts of the world the customer has a legal right to a refund under certain circumstances. DIG has a strict policy on all sales being final and no refunds

- There are also requirements as to what information must be displayed for the products offered. These include, but are not limited to system requirements, description and the inclusion of publisher provided images.

DIG does not comply with any of these!

[Comment created by bb34887: 2018-07-06]
ITAD supports DIG's bundles. How do you know it's AAA store is "unauthorized"?

And for that matter, what's wrong with buying a key legitimately and then re-selling it? Re-sale of physical media is common and not thought of as "dodgy", and it does more "harm" to publishers because physical media can be used by many consecutive owners, whereas a key can only be redeemed once.

[Comment created by KNRT: 2018-07-04]
ITAD doesn't list gray market/unauthorized resellers.

**Comment by MarkitkaM** [comment]
Seems like solved within the conversation above enough.
